posts/inside-rust/2021-01-15-rustdoc-performance-improvements.md

```diff
@@ -0,0 +1,178 @@
+---
+layout: post
+title: "Rustdoc performance improvements"
+author: Joshua Nelson and Guillaume Gomez
+team: The Rustdoc Team <https://www.rust-lang.org/governance/teams/dev-tools#rustdoc>
+---
+
+Hi everyone! [**@GuillaumeGomez**] recently tweeted about the rustdoc performance improvements and suggested that we write a blog post about it:
+
+<blockquote class="twitter-tweet"><p lang="en" dir="ltr">The performance comparison between <a href="https://twitter.com/rustlang?ref_src=twsrc%5Etfw">@rustlang</a> rustdoc now and rustdoc 4 months ago are crazy! The rustdoc cleanup going on (we&#39;re still far from done!) is having a huge positive impact! Can&#39;t wait to see how much better the results will be.<br><br>Maybe I should write a blog post? <a href="https://t.co/XapdmdZ1IZ">pic.twitter.com/XapdmdZ1IZ</a></p>&mdash; Guillaume Gomez (@imperioworld_) <a href="https://twitter.com/imperioworld_/status/1349383125051305984?ref_src=twsrc%5Etfw">January 13, 2021</a></blockquote> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script> 
+
+The tweet received a lot of comments approving the blog post idea so here we go!
+
+## Performance changes
+
+There were actually only two PRs explicitly meant to improve the performance of rustdoc.
+
+1. Rustdoc: Cache resolved links [#77700](https://github.com/rust-lang/rust/pull/77700)
+
+This does what it says in the title. In particular, this sped up the time to generate intra-doc
+links for `stm32h7xx` by a whopping [90000%]. [**@bugadani**](https://github.com/bugadani) did an
+excellent job on this, congratulations!
+
+[90000%]: https://github.com/rust-lang/rust/pull/77700#issuecomment-735995025.
```

Comment:
I think it's because it has a trailing period. Good catch!

```suggestion
[90000%]: https://github.com/rust-lang/rust/pull/77700#issuecomment-735995025
```